tools/oldest_constraints.txt

```diff
@@ -60,6 +60,7 @@ distro==1.0.1
 # Lexicon oldest constraint is overridden appropriately on relevant DNS provider plugins
 # using their local-oldest-requirements.txt
 dns-lexicon==2.2.1
+httplib2==0.9.2
```

Comment:
I think that's already done but not shown in the GitHub UI. This is pinned under a section that starts with the comment `# Ubuntu Bionic constraints`.

The reason I tackled part of https://github.com/certbot/certbot/issues/6473 and pinned this back now was due to newer versions needing a newer versions of `setuptools` than is currently available in our oldest tests, but the reason this specific version was chosen is because it's the version packaged in Ubuntu Bionic and being used by our `python3-certbot-dns-google` package there. See https://packages.ubuntu.com/bionic/python3-httplib2.

Reply:
Works for me!